app/query/processors.go

```diff
@@ -389,6 +405,57 @@ func resolve(ctx context.Context, c *Caller, q *Query, url string) (*ljsonrpc.Cl
 	return &claim, err
 }
 
+// preflightHookClaimSearch patches tag parameters of RPC request to support scheduled and unlisted content.
+func preflightHookClaimSearch(_ *Caller, ctx context.Context) (*jsonrpc.RPCResponse, error) {
+	query := GetFromContext(ctx)
+	params := query.ParamsAsMap()
+	if hasSource, ok := params["has_source"].(bool); ok && hasSource {
+		notTags, _ := params["not_tags"].([]string)
+		anyTags, _ := params["any_tags"].([]string)
+		if !hasAnyTag(params["any_tags"], ClaimTagPrivate, ClaimTagUnlisted) {
+			if !hasTag(notTags, ClaimTagPrivate) {
+				notTags = append(notTags, ClaimTagPrivate)
+			}
+			if !hasTag(notTags, ClaimTagUnlisted) {
+				notTags = append(notTags, ClaimTagUnlisted)
+			}
+			params["not_tags"] = notTags
+		}
+		if !hasTag(anyTags, ClaimTagScheduledShow) {
```

Comment:
something also isn't working properly here, it's still filtering future ones when passing the tag. This example had: 
![image](https://user-images.githubusercontent.com/8120721/234699101-87d3bd53-8e06-4307-95a8-396467822c7b.png)

I just adjusted the logic to also check for c:scheduled:hide, since we'll use that to show all future uploads. So check for either here.